### ISSUE TITLE
Bond dipole indicator location can be confusing when atoms are in close proximity

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/molecule-polarity#121]
**Test device**
MacBook Air M1 chip

**Operating System**
11.4

**Browser**
safari 14.1.1

**Problem description**
https://github.com/phetsims/qa/issues/680

In the Three Atoms screen, it is unclear which bond dipole indicator belongs to the A-B bond and which bond dipole indicator belongs to the C-B bond when A and C atoms are in close proximity. In addition, if Partial Charges is also selected in the View box, the partial charge of the B atom becomes covered by the bond dipoles. 

**Steps to reproduce**
1. Select the Three Atoms Screen
2. Turn on Bond Dipoles in the View box
3. Move the C atom counterclockwise until the bond dipole indicator for the C-B bond passes the bond dipole indicator for the A-B bond. 
4. Turn on Partial Charges in the View box
5. Slowly move the C atom clockwise until the bond dipole indicators cross over the partial charge of the B atom. 

**Visuals**
https://drive.google.com/file/d/1wrd2LtrhzW8m8_bBzY8pjimAbNKZkRbX/view?usp=sharing

<img width="866" alt="Screen Shot 2021-07-28 at 1 26 24 PM" src="https://user-images.githubusercontent.com/87318828/127369174-cfaab4c7-0059-41d9-adb0-a09020a123af.png">

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Molecule Polarity‬
URL: https://phet-dev.colorado.edu/html/molecule-polarity/1.2.0-dev.3/phet/molecule-polarity_all_phet.html
Version: 1.2.0-dev.3 2021-07-26 23:04:53 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.1.1 Safari/605.1.15
Language: en-us
Window: 1352x683
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0 (1.0)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 31 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 8192x8192
OES_texture_float: true
Dependencies JSON: {}

**Comment by pixelzoom** [comment]
@Nancy-Salpepi thanks for reporting.

>  it is unclear which bond dipole indicator belongs to the A-B bond and which bond dipole indicator belongs to the C-B bond when A and C atoms are in close proximity.

The bond dipole is always parallel to the bond, and offset a bit. This is a convention that we adopted from JSmol, a programming library that was used to implement the "Real Molecules" (3D) screen in the Java version - see first screenshot below.  Drawing the dipole parallel to the bond is also a general chemistry convention, though sometimes the dipole is drawn on top of the bond (which makes it more difficult to see).  So I don't think there's anything to be done here - the dipole parallel to its associated bond.

>  if Partial Charges is also selected in the View box, the partial charge of the B atom becomes covered by the bond dipoles.

In addition to the screenshot that you provided, there's also opportunity for overlap with the bonds - see second screenshot below.  

The position of the partial charge was discussed way back (2010?) in the design of the Java version. We decided to place the partial charge on the side of atom B that is opposite the molecular dipole. We chose a consistent place for it, because it would distracting (and potentially confusing) to have it jumping around to avoid overlap.  It's also a difficult programming problem to try to avoid overlap. So yes, there will be some cases where things overlap. But that's preferrable to having the partial charge jump around to avoid overlap.

@arouinfar let me know if you'd like to change anything here.

<img width="1024" alt="screenshot_1099" src="https://user-images.githubusercontent.com/3046552/127374995-818f48b4-e38c-4c19-b217-c1e44ef9bb06.png">

<img width="414" alt="screenshot_1098" src="https://user-images.githubusercontent.com/3046552/127373461-b4aebe90-9959-4e18-97ae-f568ad5887b0.png">
